srcpkgs/signify/template

```diff
@@ -1,27 +1,30 @@
 # Template file for 'signify'
 pkgname=signify
 reverts="20141230_3 20141230_2 20141230_1"
-version=30
+version=31
 revision=1
 build_style=gnu-makefile
 make_build_args="BZERO=bundled"
 hostmakedepends="pkg-config"
-makedepends="libbsd-devel"
+makedepends="libbsd-devel libmd-devel"
+checkdepends="tar"
 short_desc="OpenBSD cryptographic signing and verification tool"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="ISC"
-homepage="http://www.tedunangst.com/flak/post/signify"
+homepage="https://www.tedunangst.com/flak/post/signify"
+changelog="https://raw.githubusercontent.com/aperezdc/signify/master/CHANGELOG.md"
 distfiles="https://github.com/aperezdc/signify/releases/download/v${version}/signify-${version}.tar.xz"
-checksum=f68406c3085ef902e85500e6c0b90e4c3f56347e5efffc0da7b6fb47803c8686
+checksum=1155fd9eeed4a8aa20476b2333d251953ec5d52338d943a770db5b78dd8d2b74
 
 pre_build() {
 	# Allow building with musl
 		cp ${FILESDIR}/void.h .
 		for file in blf.h sha2.h ohash.h; do
-			sed -e '1 i\#include "void.h"' -i $file
+			vsed -e '1 i\#include "void.h"' -i $file
 		done
+
 	# Create license file
-		head -n 16 signify.c > LICENSE
+		sed -E '3,15 !d;s/^ [*] ?//' signify.c > LICENSE
```

Comment:
can you remove the extra tab indents here

Reply:
Done